releasing-archway.md

```diff
@@ -1,74 +1,41 @@
-    (X.Y.0-rc.W | W > 0) as necessary.
-- X.Y.0 (Branch: release-X.Y)
-  - Final release, cut from the release-X.Y branch.
-  - X.Y.0-rc.0 will be tagged at the same commit on the same branch.
-- X.Y.Z, Z > 0 (Branch: release-X.Y) ([Patch releases](#patch-releases))
-  - [Patch releases](#patch-releases) are released as we cherrypick commits from main into
-    the release-X.Y branch, as needed.
-  - X.Y.Z is cut straight from the release-X.Y branch, and X.Y.Z+1-beta.0 is
-    tagged on the followup commit.
-- X.Y.Z, Z > 0 (Branch: release-X.Y.Z) (Branched [patch releases](#patch-releases) only for hotfix situations)
-  - These are rarely used and are special in that the X.Y.Z tag is branched to isolate
-    the emergency/critical fix from all other changes that have landed on the
-    release branch since the previous tag
-  - Cut release-X.Y.Z branch to hold the isolated patch release
-  - Tag release-X.Y.Z branch + fixes with X.Y.(Z+1)
-  - Branched [patch releases](#patch-releases) are rarely needed but used for
-    emergency/critical fixes to the latest release
-
-### Major version timeline
-
-There is currently no mandated timeline for major versions beyond version 1.Y.Z, Y,Z >= 0. We haven't so far applied a rigorous interpretation of semantic
-versioning with respect to incompatible changes of any kind.
-
-TODO: Major versioning criteria need to be put up for discussion once Archway protocol reaches 1.Y.Z release.
-
-## Patch releases
-
-Patch releases are intended for critical bug fixes to the latest minor version,
-such as addressing security vulnerabilities, fixes to problems affecting a large
-number of users and severe problems with no workaround.
-
-They should not contain miscellaneous feature additions or improvements, and
-especially no incompatibilities should be introduced between patch versions of
-the same minor version (or even major version).
-
-Dependencies, such as cosmos-sdk or tendermint, should also not be changed unless
-absolutely necessary, and also just to fix critical bugs (so, at most patch
-version changes, not new major nor minor versions).
+# Archway Protocol Release Process
+
+## Release Versioning
+
+This document outlines the release process for Archway protocol software.
+
+The **Archway** protocol follows a **state versioning system**, it does not utilize traditional [semantic versioning](http://semver.org); SemVer is specifically about APIs, but API breaking changes is not the only way in which libraries can "break". In deterministic systems, such as a blockchain being deterministic state machine, *state breaking changes far outweigh API breaking changes*. As such, Archway follows a State Versioning system.
+
+Given a version number Major.Minor.Patch, increment the:
+
+1. **Major** version when any state breaking changes are introduced;
+2. **Minor** version when any API changes, both API-compatible or API-incompatible, are introduced;
```

Comment:
```suggestion
2. **Minor** version when any Query API changes, both API-compatible or API-incompatible, are introduced;
```

Any Tx API is always going to be state breaking. Only query APIs can change in non state breaking but API-incomptaible ways. 

Additionally, changes to events which are emitted, as well as CLI changes should also be minor bumps.